search/roadmap.adoc

```diff
@@ -26,34 +26,46 @@ Analyzers::
 A new DSL will be introduced to allow people to define Analyzers programmatically at boostrap, as an alternative to the current `@Analyzer` and `@AnalyzerDef` annotations.
 We might want to automatically register a set of commonly used Analyzers - let us know what you think of this as the choice is hard!
 
+Java 9 compatibibility::
+We're aiming at regularly testing compatibility with Java 9 on top of the regular tests on Java 8.
+Testing will be limited to having Hibernate Search on the regular classpath: running as a module is an additional step we deferred to version 5.9.
+
+
+== Hibernate Search 5.9
+
+Elasticsearch performance::
+We applied some interesting performance improvements in version 5.8 but there are some more interesting ideas which could provide
+a radical boost. Moved these to 5.9 as they are quite complex and need fleshing out.
+
 WildFly Swarm::
-WildFly Swarm is the new fine-grained modular version of WildFly aiming at microservices.
+http://wildfly-swarm.io/[WildFly Swarm] is the new fine-grained modular version of http://wildfly.org/[WildFly] aiming at microservices.
 We should make sure Hibernate Search is integrated properly and the usage is documented.
 
-Java 9 compatibibility::
-We're aiming at regularly testing compatibility with Java 8 and now 9 too.
-Hibernate Search should test its modularity within project Jigsaw, within reason as we can't depent on Java9 release schedules.
-
 Removal of class definitions from master nodes::
```

Comment:
Let's remove that. I don't even know where to begin, and I suspect we'll need to do the 6.0 work first. Also I think this 5.9 should only address very clearly identified topics.

Reply:
ok I'll remove this.

Reply:
BTW while I'm removing this, remember most of the work was done already ;)